src/plugins/data/common/search/aggs/param_types/field.ts

```diff
@@ -92,7 +92,7 @@ export class FieldParamType extends BaseParamType {
               defaultMessage:
                 'Saved {fieldParameter} parameter is now invalid. Please select a new field.',
               values: {
-                fieldParameter: '"field"',
+                fieldParameter: `"${fieldName}"`,
```

Comment:
Since this is targeting a minor, not a patch, we can create new translations. My preference would be a message like `Saved field "order_date" is invalid for use with the "terms" aggregation. Please select a new field.`

Reply:
@wylieconlon done. much better this way 👍  
![Discover_-_Elastic](https://user-images.githubusercontent.com/463851/94568295-9ab1d000-026c-11eb-8579-00d64695d342.png)